datasets/gov/lsd/gyventoju_uzimtumas.csv

```diff
@@ -0,0 +1,425 @@
+,,,,,,enum,,,"""01""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""02""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""03""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""04""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""05""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""06""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""07""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""08""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""09""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""10""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""11""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""12""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,,,,,"""13""",,,,"Savaitės, trunkančios nuo pirmadienio iki sekmadienio, numeris",
+,,,,,region,string,,,,4,open,,"Regionas, kuriame yra namų ūkis",
+,,,,,,enum,,,"""001""",,,,"Alytaus apskritis",
+,,,,,,,,,"""002""",,,,"Kauno apskritis",
+,,,,,,,,,"""003""",,,,"Klaipėdos apskritis",
+,,,,,,,,,"""004""",,,,"Marijampolės apskritis",
+,,,,,,,,,"""005""",,,,"Panevežio apskritis",
+,,,,,,,,,"""006""",,,,"Šiaulių apskritis",
+,,,,,,,,,"""007""",,,,"Tauragės apskritis",
+,,,,,,,,,"""008""",,,,"Telšių apskritis",
+,,,,,,,,,"""009""",,,,"Utenos apskritis",
+,,,,,,,,,"""00A""",,,,"Vilniaus apskritis",
+,,,,,hhnum,integer,,,,4,open,,"Namų ūkio eilės numeris",
+,,,,,coeff,number,,,,4,open,,Svoris,
+,,,,,intwave,integer,,,,4,open,,"Tyrimo etapo eilės numeris",
+,,,,,,enum,,,1,,,,"Dalyvauja tyrime  pirmą kartą",
+,,,,,,,,,2,,,,"Dalyvauja tyrime antrą kartą",
+,,,,,,,,,3,,,,"Dalyvauja tyrime trečią kartą",
+,,,,,,,,,4,,,,"Dalyvauja tyrime ketvirtą kartą",
+,,,,,,,,,900,,,,Nežinomas,
+,,,,,,,,,999,,,,"Netaikoma ",
+,,,,,courpurp,integer,,,,4,open,,"Pagrindinės dalyvavimo šiuose mokymuose priežastys",
+,,,,,,enum,,,1,,,,"Susijusios su darbu",
+,,,,,,,,,2,,,,Asmeninės,
+,,,,,,,,,9,,,,"Netaikoma ",
+,,,,,courfild,string,,,,4,open,,"Paskutinio mokymosi mokymo programa",
+,,,,,,enum,,,"""000""",,,,"Bendrosios programos",
+,,,,,,,,,"""100""",,,,"Mokytojų rengimas ir švietimo mokslas",
+,,,,,,,,,"""200""",,,,"Humanitariniai mokslai, kalbos ir menai",
+,,,,,,,,,"""300""",,,,"Socialiniai mokslai, verslas ir teisė",
+,,,,,,,,,"""400""",,,,"Gamtos mokslai, matematika ir naudojimasis kompiuteriu",
+,,,,,,,,,"""500""",,,,"Inžinerija, gamyba ir statyba",
+,,,,,,,,,"""600""",,,,"Žemės ūkis ir veterinarija",
+,,,,,,,,,"""700""",,,,"Sveikata ir rūpyba",
+,,,,,,,,,"""800""",,,,Paslaugos,
+,,,,,,,,,"""900""",,,,Nežinomas,
+,,,,,,,,,"""999""",,,,"Netaikoma ",
+,,,,,metai,date,Y,,,4,open,,"Statistinio tyrimo metai",
```

Comment:
Kaip suprantu, anksčiau tyrimai buvo atliekami ketvirčiais, o dabar nurodyta metais?

Reply:
yra ir dabar ketvirčiai, buvo pridėti https://github.com/atviriduomenys/manifest/pull/1384#issuecomment-1187325633
bet kazkuriame komentare sakei pridėti tik metus.. tiesa sakant su statistinių duomenų aprašinėjimais perdaug nežinomųjų, nera nei dokumentacijos, nei paaiškinimų kaip geriausia daryti..